finagle-example/src/main/java/com/twitter/finagle/example/java/http/CatService.java

```diff
@@ -0,0 +1,9 @@
+package com.twitter.finagle.example.java.http;
+
+public final class CatService {
+    static CatsDB db = new CatsDB();
```

Comment:
can we mark this `private final`? good to have good code hygiene even in sample code.